libs/platform/src/lib/components/form/input/fdp-input.module.ts

```diff
@@ -1,12 +1,22 @@
 import { NgModule } from '@angular/core';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { InputComponent } from './input.component';
 import { CommonModule } from '@angular/common';
 import { FormModule as FdFormModule } from '@fundamental-ngx/core';
-import { InputComponent } from './input.component';
+import { FormsModule as coreFormModule} from '@angular/forms';
+import { FdpFormGroupModule } from '../form-group/fdp-form.module';
 
 @NgModule({
-    declarations: [InputComponent],
-    imports: [CommonModule, FormsModule, ReactiveFormsModule, FdFormModule],
-    exports: [InputComponent]
+    declarations: [
+        InputComponent
+    ],
+    imports: [
+        CommonModule,
+        FdpFormGroupModule,
```

Comment:
There should be no reference to input module. FormGroup, FormField should know nothing about specific controls.